libs/ui/tabs.jsx

```diff
@@ -187,14 +187,23 @@ export function TabPanels (props) {
                   animationInDuration={800}
                   animationOutDuration={800}
                   isVisible={!props.hideDataTable}>
-                  <DataGrid
-                    ref={props.dataGridRef}
-                    columns={props.dataColumns}
-                    rows={props.data}
-                    enableCellAutoFocus={false}
-                    height={props.topRightPaneHeight}
-                    onGridSort={props.sortRows}
-                    />
+                  {props.dataColumns.length > 0 ?
+                    <DataGrid
+                      ref={props.dataGridRef}
+                      columns={props.dataColumns}
+                      rows={props.data}
+                      enableCellAutoFocus={false}
+                      height={props.topRightPaneHeight}
+                      onGridSort={props.sortRows}
+                      />
+                    :
+                      <div className="emptyPanelWrapper">
+                        <FontAwesomeIcon className="panelIcon"icon={faDatabase} />
+                        <p className="emptyPanelText">You aren't currently displaying any data.</p>
+                        <p className="emptyPanelText"> To learn more about getting started visit&nbsp;<a href="/guide/" className="guideLink">our guide</a>.
```

Comment:
The absolute URL `/guide/` won't resolve for international versions: I think it wants to be `./guide/` to be relative to the language.